src/conduit-lwt-unix/conduit_lwt_server.mli

```diff
@@ -10,6 +10,7 @@ val process_accept :
   unit Lwt.t
 
 val init :
```

Comment:
I should probably document this, I was first waiting to see if you are interested at all and what the review says about the implementation